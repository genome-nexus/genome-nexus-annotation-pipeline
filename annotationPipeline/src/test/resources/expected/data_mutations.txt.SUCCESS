#genome_nexus_version: 0-unknown-version-SNAPSHOT
#isoform: uniprot
#version 2.4
Hugo_Symbol	Entrez_Gene_Id	Center	NCBI_Build	Chromosome	Start_Position	End_Position	Strand	Consequence	Variant_Classification	Variant_Type	Reference_Allele	Tumor_Seq_Allele1	Tumor_Seq_Allele2	dbSNP_RS	dbSNP_Val_Status	Tumor_Sample_Barcode	Matched_Norm_Sample_Barcode	Match_Norm_Seq_Allele1	Match_Norm_Seq_Allele2	Tumor_Validation_Allele1	Tumor_Validation_Allele2	Match_Norm_Validation_Allele1	Match_Norm_Validation_Allele2	Verification_Status	Validation_Status	Mutation_Status	Sequencing_Phase	Sequence_Source	Validation_Method	Score	BAM_File	Sequencer	t_ref_count	t_alt_count	n_ref_count	n_alt_count	HGVSc	HGVSp	HGVSp_Short	Transcript_ID	RefSeq	Protein_position	Codons	Exon_Number	AA_AF	AF	AFR_AF	ALLELE_NUM	AMR_AF	ASN_AF	Allele	Amino_acids	BIOTYPE	CANONICAL	CCDS	CDS_position	CLIN_SIG	DISTANCE	DOMAINS	EAS_AF	EA_AF	ENSP	EUR_AF	EXON	ExAC_AC_AN	ExAC_AC_AN_AFR	ExAC_AC_AN_AMR	ExAC_AC_AN_Adj	ExAC_AC_AN_EAS	ExAC_AC_AN_FIN	ExAC_AC_AN_NFE	ExAC_AC_AN_OTH	ExAC_AC_AN_SAS	ExAC_AF	ExAC_AF_AFR	ExAC_AF_AMR	ExAC_AF_Adj	ExAC_AF_EAS	ExAC_AF_FIN	ExAC_AF_NFE	ExAC_AF_OTH	ExAC_AF_SAS	ExAC_FILTER	Existing_variation	FILTER	Feature	Feature_type	GENE_PHENO	Gene	HGNC_ID	HGVS_OFFSET	HIGH_INF_POS	IMPACT	INTRON	MINIMISED	MOTIF_NAME	MOTIF_POS	MOTIF_SCORE_CHANGE	PHENO	PICK	PUBMED	PolyPhen	SAS_AF	SIFT	SOMATIC	STRAND_VEP	SWISSPROT	SYMBOL	SYMBOL_SOURCE	TREMBL	TSL	UNIPARC	VARIANT_CLASS	all_effects	cDNA_position	flanking_bps	gnomAD_AF	gnomAD_AFR_AF	gnomAD_AMR_AF	gnomAD_ASJ_AF	gnomAD_EAS_AF	gnomAD_FIN_AF	gnomAD_NFE_AF	gnomAD_OTH_AF	gnomAD_SAS_AF	n_depth	t_depth	vcf_id	vcf_pos	vcf_qual	Annotation_Status
AL645728.3		.	GRCh37	1	1517171	1517171	+	downstream_gene_variant	3'Flank	SNP	G	G	A	rs753543553		7316-918	BS_84QXT7FR	G	G														34	7	17	0				ENST00000426195		96/634				0.1779	0.2156	1	0.1282		A	R/Q	protein_coding	YES	CCDS31.1	287/1905			Coiled-coils_(Ncoils):Coil,hmmpanther:PTHR23075,hmmpanther:PTHR23075:SF4	0.3323		ENSP00000368030	0.004	3/16																				rs1619925,COSM4596874	PASS	ENST00000378755	Transcript	1	ENSG00000197785	HGNC:25567			MODERATE						0,1	1		benign(0)	0.182	tolerated_low_confidence(0.58)	0,1	1	Q9NVI7	ATAD3A	HGNC		2	UPI000013D456	SNV	ATAD3A,missense_variant,p.Arg96Gln,ENST00000378755,NM_018188.3;ATAD3A,intron_variant,,ENST00000339113,;ATAD3A,intron_variant,,ENST00000378756,NM_001170535.1;ATAD3A,intron_variant,,ENST00000536055,NM_001170536.1;ATAD3A,non_coding_transcript_exon_variant,,ENST00000439513,;ATAD3A,upstream_gene_variant,,ENST00000429957,;	381/2612	CGG	0.03914	0.1307	0.04243	0.0006101	0.2592	0.01968	0.0008531	0.02897	0.04709	17	41	.	1517171	.	SUCCESS
