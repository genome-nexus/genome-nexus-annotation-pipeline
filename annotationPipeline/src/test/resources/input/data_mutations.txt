#version 2.4
Hugo_Symbol	Center	NCBI_Build	Chromosome	Start_Position	End_Position	Strand	Variant_Classification	Variant_Type	Reference_Allele	Tumor_Seq_Allele1	Tumor_Seq_Allele2	dbSNP_RS	dbSNP_Val_Status	Tumor_Sample_Barcode	Matched_Norm_Sample_Barcode	Match_Norm_Seq_Allele1	Match_Norm_Seq_Allele2	Tumor_Validation_Allele1	Tumor_Validation_Allele2	Match_Norm_Validation_Allele1	Match_Norm_Validation_Allele2	Verification_Status	Validation_Status	Mutation_Status	Sequencing_Phase	Sequence_Source	Validation_Method	Score	BAM_File	Sequencer	Tumor_Sample_UUID	Matched_Norm_Sample_UUID	HGVSc	HGVSp	HGVSp_Short	Transcript_ID	Exon_Number	t_depth	t_ref_count	t_alt_count	n_depth	n_ref_count	n_alt_count	all_effects	Allele	Gene	Feature	Feature_type	Consequence	cDNA_position	CDS_position	Protein_position	Amino_acids	Codons	Existing_variation	ALLELE_NUM	DISTANCE	STRAND_VEP	SYMBOL	SYMBOL_SOURCE	HGNC_ID	BIOTYPE	CANONICAL	CCDS	ENSP	SWISSPROT	TREMBL	UNIPARC	RefSeq	SIFT	PolyPhen	EXON	INTRON	DOMAINS	AF	AFR_AF	AMR_AF	ASN_AF	EAS_AF	EUR_AF	SAS_AF	AA_AF	EA_AF	CLIN_SIG	SOMATIC	PUBMED	MOTIF_NAME	MOTIF_POS	HIGH_INF_POS	MOTIF_SCORE_CHANGE	IMPACT	PICK	VARIANT_CLASS	TSL	HGVS_OFFSET	PHENO	MINIMISED	ExAC_AF	ExAC_AF_AFR	ExAC_AF_AMR	ExAC_AF_EAS	ExAC_AF_FIN	ExAC_AF_NFE	ExAC_AF_OTH	ExAC_AF_SAS	GENE_PHENO	FILTER	flanking_bps	vcf_id	vcf_qual	ExAC_AF_Adj	ExAC_AC_AN_Adj	ExAC_AC_AN	ExAC_AC_AN_AFR	ExAC_AC_AN_AMR	ExAC_AC_AN_EAS	ExAC_AC_AN_FIN	ExAC_AC_AN_NFE	ExAC_AC_AN_OTH	ExAC_AC_AN_SAS	ExAC_FILTER	gnomAD_AF	gnomAD_AFR_AF	gnomAD_AMR_AF	gnomAD_ASJ_AF	gnomAD_EAS_AF	gnomAD_FIN_AF	gnomAD_NFE_AF	gnomAD_OTH_AF	gnomAD_SAS_AF	vcf_pos
ATAD3A	.	GRCh38	chr1	1517171	1517171	+	Missense_Mutation	SNP	G	G	A	rs1619925		7316-918	BS_84QXT7FR	G	G																c.287G>A	p.Arg96Gln	p.R96Q	ENST00000378755	3/16	41	34	7	17	17	0	ATAD3A,missense_variant,p.Arg96Gln,ENST00000378755,NM_018188.3;ATAD3A,intron_variant,,ENST00000339113,;ATAD3A,intron_variant,,ENST00000378756,NM_001170535.1;ATAD3A,intron_variant,,ENST00000536055,NM_001170536.1;ATAD3A,non_coding_transcript_exon_variant,,ENST00000439513,;ATAD3A,upstream_gene_variant,,ENST00000429957,;	A	ENSG00000197785	ENST00000378755	Transcript	missense_variant	381/2612	287/1905	96/634	R/Q	cGg/cAg	rs1619925,COSM4596874	1		1	ATAD3A	HGNC	HGNC:25567	protein_coding	YES	CCDS31.1	ENSP00000368030	Q9NVI7		UPI000013D456	NM_018188.3	tolerated_low_confidence(0.58)	benign(0)	3/16		Coiled-coils_(Ncoils):Coil,hmmpanther:PTHR23075,hmmpanther:PTHR23075:SF4	0.1779	0.2156	0.1282		0.3323	0.004	0.182				0,1						MODERATE	1	SNV	2		0,1										1	PASS	CGG	.	.												0.03914	0.1307	0.04243	0.0006101	0.2592	0.01968	0.0008531	0.02897	0.04709	1517171
CA6	.	GRCh38	chr1	8958917	8958917	+	Missense_Mutation	SNP	T	T	C	novel		7316-918	BS_84QXT7FR	T	T																c.416T>C	p.Ile139Thr	p.I139T	ENST00000377436	4/8	70	59	11	23	22	1	CA6,missense_variant,p.Ile139Thr,ENST00000377443,NM_001215.3;CA6,missense_variant,p.Ile139Thr,ENST00000377436,NM_001270500.1;CA6,missense_variant,p.Ile79Thr,ENST00000377442,NM_001270501.1,NM_001270502.1;CA6,missense_variant,p.Ile107Thr,ENST00000549778,;CA6,non_coding_transcript_exon_variant,,ENST00000476083,;	C	ENSG00000131686	ENST00000377436	Transcript	missense_variant	416/942	416/942	139/313	I/T	aTt/aCt		1		1	CA6	HGNC	HGNC:1380	protein_coding	YES	CCDS57970.1	ENSP00000366654	P23280		UPI00004CA0CB	NM_001270500.1	deleterious(0)	benign(0.368)	4/8		PROSITE_profiles:PS51144,cd03125,hmmpanther:PTHR18952,hmmpanther:PTHR18952:SF110,PROSITE_patterns:PS00162,Gene3D:3.10.200.10,Pfam_domain:PF00194,SMART_domains:SM01057,Superfamily_domains:SSF51069																	MODERATE	1	SNV	1													PASS	ATT	.	.																					8958917
MAP3K6	.	GRCh38	chr1	27361345	27361345	+	Splice_Site	SNP	C	C	A	rs771659002		7316-918	BS_84QXT7FR	C	C																c.1736+1G>T		p.X579_splice	ENST00000493901		78	59	19	28	28	0	MAP3K6,splice_donor_variant,,ENST00000357582,;MAP3K6,splice_donor_variant,,ENST00000374040,NM_001297609.1;MAP3K6,splice_donor_variant,,ENST00000472410,;MAP3K6,splice_donor_variant,,ENST00000493901,NM_004672.4;MAP3K6,upstream_gene_variant,,ENST00000486046,;MAP3K6,upstream_gene_variant,,ENST00000470890,;MAP3K6,upstream_gene_variant,,ENST00000476509,;MAP3K6,upstream_gene_variant,,ENST00000495230,;,regulatory_region_variant,,ENSR00000003702,;	A	ENSG00000142733	ENST00000493901	Transcript	splice_donor_variant						rs771659002	1		-1	MAP3K6	HGNC	HGNC:6858	protein_coding	YES	CCDS299.1	ENSP00000419591	O95382		UPI0000205587	NM_004672.4				13/29																		HIGH	1	SNV	5												1	PASS	ACC	.	.												4.073e-06						8.989e-06			27361345
SPAG17	.	GRCh38	chr1	118066887	118066887	+	Missense_Mutation	SNP	C	C	T	novel		7316-918	BS_84QXT7FR	C	C																c.2398G>A	p.Ala800Thr	p.A800T	ENST00000336338	18/49	70	55	15	29	29	0	SPAG17,missense_variant,p.Ala800Thr,ENST00000336338,NM_206996.2;SPAG17,intron_variant,,ENST00000477444,;	T	ENSG00000155761	ENST00000336338	Transcript	missense_variant	2464/6924	2398/6672	800/2223	A/T	Gcc/Acc		1		-1	SPAG17	HGNC	HGNC:26620	protein_coding	YES	CCDS899.1	ENSP00000337804	Q6Q759		UPI00001601FD	NM_206996.2	deleterious(0)	probably_damaging(0.998)	18/49		hmmpanther:PTHR21963																	MODERATE	1	SNV	1													PASS	GCT	.	.																					118066887
IGFN1	.	GRCh38	chr1	201208915	201208915	+	Missense_Mutation	SNP	A	A	G	rs1247701351		7316-918	BS_84QXT7FR	A	A																c.4022A>G	p.Asp1341Gly	p.D1341G	ENST00000335211	12/24	73	53	13	27	24	0	IGFN1,missense_variant,p.Asp1341Gly,ENST00000335211,NM_001164586.1;IGFN1,intron_variant,,ENST00000295591,;IGFN1,upstream_gene_variant,,ENST00000412892,;IGFN1,intron_variant,,ENST00000437879,;IGFN1,downstream_gene_variant,,ENST00000444705,;IGFN1,upstream_gene_variant,,ENST00000473483,;	G	ENSG00000163395	ENST00000335211	Transcript	missense_variant	4152/11810	4022/11127	1341/3708	D/G	gAt/gGt	rs1247701351	1		1	IGFN1	HGNC	HGNC:24607	protein_coding	YES	CCDS53455.1	ENSP00000334714	Q86VF2		UPI0001B300F4	NM_001164586.1	tolerated(1)	possibly_damaging(0.737)	12/24		mobidb-lite																	MODERATE	1	SNV	5													PASS	GAT	.	.																					201208915
