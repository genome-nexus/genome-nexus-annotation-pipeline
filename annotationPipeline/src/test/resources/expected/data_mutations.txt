#genome_nexus_version: 0-unknown-version-SNAPSHOT
#isoform: uniprot
#version 2.4
Hugo_Symbol	Entrez_Gene_Id	Center	NCBI_Build	Chromosome	Start_Position	End_Position	Strand	Consequence	Variant_Classification	Variant_Type	Reference_Allele	Tumor_Seq_Allele1	Tumor_Seq_Allele2	dbSNP_RS	dbSNP_Val_Status	Tumor_Sample_Barcode	Matched_Norm_Sample_Barcode	Match_Norm_Seq_Allele1	Match_Norm_Seq_Allele2	Tumor_Validation_Allele1	Tumor_Validation_Allele2	Match_Norm_Validation_Allele1	Match_Norm_Validation_Allele2	Verification_Status	Validation_Status	Mutation_Status	Sequencing_Phase	Sequence_Source	Validation_Method	Score	BAM_File	Sequencer	t_ref_count	t_alt_count	n_ref_count	n_alt_count	HGVSc	HGVSp	HGVSp_Short	Transcript_ID	RefSeq	Protein_position	Codons	Exon_Number	AA_AF	AF	AFR_AF	ALLELE_NUM	AMR_AF	ASN_AF	Allele	Amino_acids	BIOTYPE	CANONICAL	CCDS	CDS_position	CLIN_SIG	DISTANCE	DOMAINS	EAS_AF	EA_AF	ENSP	EUR_AF	EXON	ExAC_AC_AN	ExAC_AC_AN_AFR	ExAC_AC_AN_AMR	ExAC_AC_AN_Adj	ExAC_AC_AN_EAS	ExAC_AC_AN_FIN	ExAC_AC_AN_NFE	ExAC_AC_AN_OTH	ExAC_AC_AN_SAS	ExAC_AF	ExAC_AF_AFR	ExAC_AF_AMR	ExAC_AF_Adj	ExAC_AF_EAS	ExAC_AF_FIN	ExAC_AF_NFE	ExAC_AF_OTH	ExAC_AF_SAS	ExAC_FILTER	Existing_variation	FILTER	Feature	Feature_type	GENE_PHENO	Gene	HGNC_ID	HGVS_OFFSET	HIGH_INF_POS	IMPACT	INTRON	MINIMISED	MOTIF_NAME	MOTIF_POS	MOTIF_SCORE_CHANGE	PHENO	PICK	PUBMED	PolyPhen	SAS_AF	SIFT	SOMATIC	STRAND_VEP	SWISSPROT	SYMBOL	SYMBOL_SOURCE	TREMBL	TSL	UNIPARC	VARIANT_CLASS	all_effects	cDNA_position	flanking_bps	gnomAD_AF	gnomAD_AFR_AF	gnomAD_AMR_AF	gnomAD_ASJ_AF	gnomAD_EAS_AF	gnomAD_FIN_AF	gnomAD_NFE_AF	gnomAD_OTH_AF	gnomAD_SAS_AF	n_depth	t_depth	vcf_id	vcf_pos	vcf_qual	Annotation_Status
AL645728.3		.	GRCh37	1	1517171	1517171	+	downstream_gene_variant	3'Flank	SNP	G	G	A	rs753543553		7316-918	BS_84QXT7FR	G	G														34	7	17	0				ENST00000426195		96/634				0.1779	0.2156	1	0.1282		A	R/Q	protein_coding	YES	CCDS31.1	287/1905			Coiled-coils_(Ncoils):Coil,hmmpanther:PTHR23075,hmmpanther:PTHR23075:SF4	0.3323		ENSP00000368030	0.004	3/16																				rs1619925,COSM4596874	PASS	ENST00000378755	Transcript	1	ENSG00000197785	HGNC:25567			MODERATE						0,1	1		benign(0)	0.182	tolerated_low_confidence(0.58)	0,1	1	Q9NVI7	ATAD3A	HGNC		2	UPI000013D456	SNV	ATAD3A,missense_variant,p.Arg96Gln,ENST00000378755,NM_018188.3;ATAD3A,intron_variant,,ENST00000339113,;ATAD3A,intron_variant,,ENST00000378756,NM_001170535.1;ATAD3A,intron_variant,,ENST00000536055,NM_001170536.1;ATAD3A,non_coding_transcript_exon_variant,,ENST00000439513,;ATAD3A,upstream_gene_variant,,ENST00000429957,;	381/2612	CGG	0.03914	0.1307	0.04243	0.0006101	0.2592	0.01968	0.0008531	0.02897	0.04709	17	41	.	1517171	.	SUCCESS
CA6		.	GRCh38	chr1	8958917	8958917	+	missense_variant	Missense_Mutation	SNP	T	T	C	novel		7316-918	BS_84QXT7FR	T	T														59	11	22	1	c.416T>C	p.Ile139Thr	p.I139T	ENST00000377436	NM_001270500.1	139/313	aTt/aCt					1			C	I/T	protein_coding	YES	CCDS57970.1	416/942			PROSITE_profiles:PS51144,cd03125,hmmpanther:PTHR18952,hmmpanther:PTHR18952:SF110,PROSITE_patterns:PS00162,Gene3D:3.10.200.10,Pfam_domain:PF00194,SMART_domains:SM01057,Superfamily_domains:SSF51069			ENSP00000366654		4/8																					PASS	ENST00000377436	Transcript		ENSG00000131686	HGNC:1380			MODERATE							1		benign(0.368)		deleterious(0)		1	P23280	CA6	HGNC		1	UPI00004CA0CB	SNV	CA6,missense_variant,p.Ile139Thr,ENST00000377443,NM_001215.3;CA6,missense_variant,p.Ile139Thr,ENST00000377436,NM_001270500.1;CA6,missense_variant,p.Ile79Thr,ENST00000377442,NM_001270501.1,NM_001270502.1;CA6,missense_variant,p.Ile107Thr,ENST00000549778,;CA6,non_coding_transcript_exon_variant,,ENST00000476083,;	416/942	ATT										23	70	.	8958917	.	FAILED
MAP3K6		.	GRCh38	chr1	27361345	27361345	+	splice_donor_variant	Splice_Site	SNP	C	C	A	rs771659002		7316-918	BS_84QXT7FR	C	C														59	19	28	0	c.1736+1G>T		p.X579_splice	ENST00000493901	NM_004672.4							1			A		protein_coding	YES	CCDS299.1							ENSP00000419591																						rs771659002	PASS	ENST00000493901	Transcript	1	ENSG00000142733	HGNC:6858			HIGH	13/29						1						-1	O95382	MAP3K6	HGNC		5	UPI0000205587	SNV	MAP3K6,splice_donor_variant,,ENST00000357582,;MAP3K6,splice_donor_variant,,ENST00000374040,NM_001297609.1;MAP3K6,splice_donor_variant,,ENST00000472410,;MAP3K6,splice_donor_variant,,ENST00000493901,NM_004672.4;MAP3K6,upstream_gene_variant,,ENST00000486046,;MAP3K6,upstream_gene_variant,,ENST00000470890,;MAP3K6,upstream_gene_variant,,ENST00000476509,;MAP3K6,upstream_gene_variant,,ENST00000495230,;,regulatory_region_variant,,ENSR00000003702,;		ACC	4.073e-06						8.989e-06			28	78	.	27361345	.	FAILED
SPAG17		.	GRCh38	chr1	118066887	118066887	+	missense_variant	Missense_Mutation	SNP	C	C	T	novel		7316-918	BS_84QXT7FR	C	C														55	15	29	0	c.2398G>A	p.Ala800Thr	p.A800T	ENST00000336338	NM_206996.2	800/2223	Gcc/Acc					1			T	A/T	protein_coding	YES	CCDS899.1	2398/6672			hmmpanther:PTHR21963			ENSP00000337804		18/49																					PASS	ENST00000336338	Transcript		ENSG00000155761	HGNC:26620			MODERATE							1		probably_damaging(0.998)		deleterious(0)		-1	Q6Q759	SPAG17	HGNC		1	UPI00001601FD	SNV	SPAG17,missense_variant,p.Ala800Thr,ENST00000336338,NM_206996.2;SPAG17,intron_variant,,ENST00000477444,;	2464/6924	GCT										29	70	.	118066887	.	FAILED
IGFN1		.	GRCh38	chr1	201208915	201208915	+	missense_variant	Missense_Mutation	SNP	A	A	G	rs1247701351		7316-918	BS_84QXT7FR	A	A														53	13	24	0	c.4022A>G	p.Asp1341Gly	p.D1341G	ENST00000335211	NM_001164586.1	1341/3708	gAt/gGt					1			G	D/G	protein_coding	YES	CCDS53455.1	4022/11127			mobidb-lite			ENSP00000334714		12/24																				rs1247701351	PASS	ENST00000335211	Transcript		ENSG00000163395	HGNC:24607			MODERATE							1		possibly_damaging(0.737)		tolerated(1)		1	Q86VF2	IGFN1	HGNC		5	UPI0001B300F4	SNV	IGFN1,missense_variant,p.Asp1341Gly,ENST00000335211,NM_001164586.1;IGFN1,intron_variant,,ENST00000295591,;IGFN1,upstream_gene_variant,,ENST00000412892,;IGFN1,intron_variant,,ENST00000437879,;IGFN1,downstream_gene_variant,,ENST00000444705,;IGFN1,upstream_gene_variant,,ENST00000473483,;	4152/11810	GAT										27	73	.	201208915	.	FAILED
